### PR TITLE
Fix point cloud ir

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

**Comment by dNechita** [comment]
Can you please add a commit that contains only the changes related to formatting?

File: examples/tof-viewer/src/ADIMainWindow.cpp [hunk 1]
```diff
@@ -1403,9 +1403,11 @@ void ADIMainWindow::PlayCCD(int modeSelect, int viewSelect) {
 
             initOpenGLPointCloudTexture();
             initOpenGLDepthTexture();
+            initOpenGLIRTexture();
         }
         if (modeSelectChanged != modeSelect) {
             view->m_ctrl->StopCapture();
+            Sleep(500);
```
Comment (dNechita) [inline examples/tof-viewer/src/ADIMainWindow.cpp:1410]: Why wait here? Why 500 milliseconds?

Comment (SeptimiuVana) [reply examples/tof-viewer/src/ADIMainWindow.cpp:1410]: This was the sleep value set when stopping the camera on depth/ab mode, I thought we need it here as well.

Comment (dNechita) [reply examples/tof-viewer/src/ADIMainWindow.cpp:1410]: If your not sure this is required, can you at least leave a to-do comment that says to check if this is required?

Comment (SeptimiuVana) [reply examples/tof-viewer/src/ADIMainWindow.cpp:1410]: Yes, sorry, I will fix this.

**Pull request merged**